src/js/utils/SchemaFormUtil.js

```diff
@@ -194,7 +194,8 @@ let SchemaFormUtil = {
 
   validateModelWithSchema(model, schema) {
     let result = tv4.validateMultiple(model, schema);
-    if (result.valid) {
+
+    if (result == null || result.valid) {
```

Comment:
we need to do the `== null` check for tests. it has no significance for the app because tv4 will always return something